modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -252,12 +286,69 @@ function AddFDSFilterModalContent({
 					value: String(item.id),
 				}));
 
+	const inUseFields = fields.map((item) =>
+		fieldNames?.includes(item.name) ? item.name : undefined
+	);
+
+	const CellRendererDropdown = ({
```

Comment:
Since this is only used once, there is no need to declare it as it's own function

Reply:
The solution comes from [Fields](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx), which follows the same pattern. If we change it here, we should do the same there to keep consistency, I think.

Reply:
Yeah, it looks like that pattern is [worse than I thought](https://github.com/liferay-frontend/liferay-portal/pull/3639#issuecomment-1713681170).